pkg/cmd/run_test.go

```diff
@@ -677,69 +677,3 @@ func TestMissingTrait(t *testing.T) {
 	assert.Equal(t, "Error: bogus.fail=i-must-fail is not a valid trait property\n", output)
 	assert.NotNil(t, err)
 }
-
-func TestGetsPaths(t *testing.T) {
```

Comment:
Out of curiosity, why is this test removed?

Reply:
Hi @tadayosi ! Thanks for reviewing! I removed it because it's no longer used. Now with the URI style options the extra ':' in windows doesn't create a special use case as the target path in the running container is specified by `?targetPath=<path>` and not `:path`

Reply:
Ah I see. Thanks for your explanation.